README.md

```diff
@@ -225,4 +225,16 @@ Python projects, questions are welcome on the scipy-user mailing
 list. Specialized discussions or design issues should take place on
```

Comment:
can you remove the mention of scipy-user on the line above?

I would make the logic like "general questions are welcome on the pydata mailing list, specialized discussion related to pandas development on pandas-dev"